README.adoc

```diff
@@ -48,6 +48,8 @@ The Vulkan Guide can be built as a single page using `asciidoctor guide.adoc`
 
 == xref:{chapters}versions.adoc[Versions]
 
+== xref:{chapters}vulkan_versions_porting_guide.adoc[Vulkan Versions & Porting Guide]
```

Comment:
need to add to `guide.adoc` as well... ya, if you have a way to have README.adoc and guide.adoc to be the same that would be a better world

Reply:
...  I'll spend time working on a CI step to automate that.  sed and regex is a powerful combo.

Reply:
The profiles PR has a script in it and it's in the CI now too.  If we detect a chapter, it auto adds it to the three files on its own if it isn't already there.

Reply:
if you want to pull that CI logic out in another PR, we can merge it in ASAP

Reply:
Sure, I can do that.